src/main.py

```diff
@@ -415,7 +400,6 @@ def process_response(response):
 			for directive in j['messageBody']['directives']:
 				if directive['namespace'] == 'SpeechSynthesizer':
 					if directive['name'] == 'speak':
-						platform.indicate_recording(False)
```

Comment:
It was removed in 852a572 cleanup and due to an incorrect rebase appeared in the mentioned commit. The logging stuff is also due to an incorrect rebase.
I'll make the change when I have the time. Thanks.